### PR TITLE
[UMF] Fix -Wempty-body warning

### PR DESCRIPTION
**Pull request by veselypeta**
When updating the tag of UR in SYCL I got the -Wempty-body warning emitted from UMF. This PR just places braces around the do statement to prevent emitting this warning.

### CODE REVIEW

**Comment by kbenzie** [comment]
> When updating the tag of UR in SYCL I got the -Wempty-body warning emitted from UMF. This PR just places braces around the do statement to prevent emitting this warning.

I feel like this patch should also enable the `-Wempty-body` warning.

**Review by igchor — APPROVED** [review]

File: cmake/helpers.cmake [hunk 1]
```diff
@@ -63,6 +63,8 @@ function(add_ur_target_compile_options name)
             -fPIC
             -Wall
             -Wpedantic
+            -Wempty-body
+            -Wunused-parameter
```
Comment (kbenzie) [inline cmake/helpers.cmake:67]: While I think adding `-Wunused-paramter` is a good idea, perhaps that should be a separate PR since its exposed additional warnings.

**Review by kbenzie — APPROVED** [review]

**Review by pbalcer — APPROVED** [review]
